src/android-verify.js

```diff
@@ -218,14 +236,19 @@ function verifyResponseSignature(response, signature, pemLeafCert, whitelistedPu
       i += 1;
     }
   } else {
+    console.log("verifyResponse SIG in else ")
     sig = new r.crypto.Signature({alg: hashAlg});
     sig.init(pemLeafCert);
     sig.updateString(response);
     result = sig.verify(signature.toString('hex'));
+    console.log("AFTER SET RESULT:",  result)
+    return result
   }
 
   if (!result) {
-    throw new Error('verifyResponseSignature failed');
+    console.log("if result not there")
+    //throw new Error('verifyResponseSignature failed');
```

Comment:
if code reaches this point, it should return false, so no need for the result check imo

Reply:
Yes, will change.